ArcadiaParties.Data.Abstractions/DTOs/UserDTO.cs

```diff
@@ -6,14 +6,12 @@ public class UserDTO
     {
         public string Identity { get; set; }
 
-        public string FirstName { get; set; }
-
-        public string LastName { get; set; }
+        public string Name { get; set; }
 
         public ICollection<CelebratingDateDTO> Dates { get; set; }
 
         public IEnumerable<string> Roles { get; set; }
 
-        public DepartmentDTO Department { get; set; }
+        public string DepartmentId { get; set; }
```

Comment:
And how user's department should be shown in user profile view in SPA?

Reply:
Fixed